admin/users.php

```diff
@@ -167,8 +167,8 @@
         }
 
         /* Attempt to create the user */
-        $user_id = User::create($username, $fullname, $email, $website, $pass1, (string) $access, $state, $city);
-        if (!$user_id) {
+        $user_id = (int) User::create($username, $fullname, $email, $website, $pass1, (string) $access, $state, $city);
```

Comment:
Any reason this needs to be casted?

Reply:
Construct for user has an int parameter for id. And create returns a string.

There are still a few inconsistencies between ids with strings and ints that need looking at

Reply:
Can't create just return an int?

Reply:
I do think that everything id related should be int. That's what we store these as.

You did say it was going to take a while. ;)